packages/package/test/fixtures/svelte-3-types/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
 	"compilerOptions": {
 		"target": "ESNext",
-		"module": "ESNext"
+		"module": "ESNext",
+		"verbatimModuleSyntax": true
```

Comment:
curious: why was this needed?

Reply:
Just to remove a warning while running the unit tests, smth like "tsconfig verbatimModuleSyntax is required when using Svelte files with lang=ts"

This is not related to the original goal of the PR, it can be reverted